MvvmCross-Forms/MvvmCross.Forms/Views/Attributes/MvxNavigationPagePresentationAttribute.cs

```diff
@@ -9,5 +9,15 @@ public class MvxNavigationPagePresentationAttribute : MvxBasePresentationAttribu
         public MvxNavigationPagePresentationAttribute()
         {
         }
+
+        public virtual bool WrapInNavigationPage { get; set; } = false;
```

Comment:
Is it possible to wrap a NavigationPage in a NavigationPage?

Reply:
Yes it is. You can build nested navigations.